src/main/PDF/PdfController.java

```diff
@@ -51,15 +38,27 @@ public PdfController(MongoDatabase db) {
               "ListBox",
               "TextField",
               "SignatureField"));
+
+  /*
+  REQUIRES JSON Body with:
+    - "pdfType": String giving PDF Type ("FORM", "APPLICATION", "IDENTIFICATION")
+    - "fileId": String giving id of file to be deleted
+  */
   public Handler pdfDelete =
       ctx -> {
-        String user = ctx.sessionAttribute("username");
-        String organizationName = ctx.sessionAttribute("orgName");
-        UserType privilegeLevel = ctx.sessionAttribute("privilegeLevel");
+        String user = Objects.requireNonNull(ctx.sessionAttribute("username"));
```

Comment:
What happens to the response when this is null? Does the frontend handle this?

Reply:
Please see above comment, we need to discuss

Reply:
you should call it username first of all, and you should do a manual if check.

Reply:
Could the server team do the error checking? We need a lot of work on server-client communications and error checking, it is a big project